kartierung_von_ueberflutungsgebieten_mit_actinia/notebooks/actinia_S1_Wasser.ipynb

```diff
@@ -615,7 +615,7 @@
     "           \"value\": \"S1_20170330_8BDD_VV_clip\"\n",
     "          },\n",
     "          {\"param\": \"mapsetid\",\n",
-    "           \"value\": \"s1watermask_tmp\"\n",
+    "           \"value\": \"S1_watermask\"\n",
```

Comment:
The mapset specified with `mapsetid` is not supposed to exist, it is created by `i.sentinel_1.water.worker`. If it exists already, it will be deleted first and then created anew. Therefore the existing mapset `S1_watermask` with  the needed input data can not be used as value for `mapsetid`.

Reply:
The current user has no permissions to create a new mapset in a process chain.